internal/policy/verify.go

```diff
@@ -211,146 +214,149 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 
 				slog.Debug("Updating current policy...")
 				currentPolicy = newPolicy
-				continue
-			}
-
-			slog.Debug("Checking if entry is for attestations reference...")
-			if entry.RefName == attestations.Ref {
+			case attestations.Ref:
+				slog.Debug("Entry is for the attestations reference...")
```

Comment:
trailing ...

Reply:
Also, the log message for this block should match the policy block above for consistency; my mistake originally. Could you add some similar statements?